test/e2e/tcp/connectivity.go

```diff
@@ -33,7 +33,10 @@ type EndpointType int
 const (
 	PodIP EndpointType = iota
 	ServiceIP
+	// TODO: Remove GlobalIP once all consumer code switches to GlobalServiceIP
 	GlobalIP
+	GlobalPodIP
```

Comment:
But GlobalPodIP is not used anywhere yet, right?

Reply:
@vthapar will be adding e2e tests in Submariner pod for GlobalPodIP once this is merged.